graylog2-server/src/main/java/org/graylog2/rest/resources/system/indexer/IndexerOverviewResource.java

```diff
@@ -169,6 +168,7 @@ private IndexSummary buildIndexSummary(Map.Entry<String, JsonNode> indexStats,
         final long deleted = docs.path("deleted").asLong();
         final JsonNode store = primaries.path("store");
         final long sizeInBytes = store.path("size_in_bytes").asLong();
+        final long shardCount = indicesAdapter.getShardsInfo(indexName).size();
```

Comment:
It's a long here and Long in the DTO. Is there any valid situation where the value is null and we need the boxed object?

Reply:
I made it @Nullable just to be on the safe side for back-compat.

Reply:
What would then be a situation which leads to a null value here, in a backwards-compatible case?

Reply:
If it were stored in the DB and you upgrade. 
Looking more closely, we never store this object in the DB. I removed the @Nullable and changed the type.